README.md

```diff
@@ -4,8 +4,8 @@
 
 This guide describes how to work inside the PX4 system architecture. It enables developers to:
 
-* Get an [overview of the system](en/setup/config_initial.md)
-* Access and modify the [PX4 Flight Stack](en/concept/flight_stack.md) and [PX4 Middleware](en/concept/middleware.md)
+* Get a [minimum developer setup](en/setup/config_initial.md).
```

Comment:
FYI, explain what it actually does. There doesn't seem to be a good "overall architecture" doc, so I suspect that is covered by "concepts". 

@bkueng Do you think we should have a bullet "understand the main PX4 concepts" pointing to that section?

Reply:
I agree the "overall architecture" page is still missing, so I'd leave it as is for now.

Reply:
OK, thank you. I'll leave that for now.